gtsam/linear/Errors.h

```diff
@@ -20,59 +20,54 @@
 #pragma once
 
 #include <gtsam/base/FastList.h>
-#include <gtsam/base/Vector.h>
 #include <gtsam/base/Testable.h>
+#include <gtsam/base/Vector.h>
 
 #include <string>
 
 namespace gtsam {
 
-  // Forward declarations
-  class VectorValues;
-
-  /** vector of errors */
-  class Errors : public FastList<Vector> {
-
-  public:
-
-    GTSAM_EXPORT Errors() ;
-
-    /** break V into pieces according to its start indices */
-    GTSAM_EXPORT Errors(const VectorValues&V);
+// Forward declarations
+class VectorValues;
 
-    /** print */
-    GTSAM_EXPORT void print(const std::string& s = "Errors") const;
+/// Errors is a vector of errors.
+using Errors = FastList<Vector>;
 
-    /** equals, for unit testing */
-    GTSAM_EXPORT bool equals(const Errors& expected, double tol=1e-9) const;
+/// Break V into pieces according to its start indices.
+GTSAM_EXPORT Errors createErrors(const VectorValues& V);
 
-    /** Addition */
-    GTSAM_EXPORT Errors operator+(const Errors& b) const;
+/// Print an Errors instance.
+GTSAM_EXPORT void print(const Errors& e, const std::string& s = "Errors");
 
-    /** subtraction */
-    GTSAM_EXPORT Errors operator-(const Errors& b) const;
+// Check equality for unit testing.
+GTSAM_EXPORT bool equality(const Errors& actual, const Errors& expected,
+                           double tol = 1e-9);
```

Comment:
These three functions break backward compatibility for people not using `traits`, though I'm sure you were already aware.